### PR TITLE
Add Amplitude chain to Kusama logos and chain info

### PR DESCRIPTION
**Pull request by gonzamontiel**
The main purpose is to be able to see our chain's name and logo for the Kusama crowdloan, parathread, and parachain pages.

Screenshot from 2022-06-29 17-39-44

### CODE REVIEW

**Comment by prayagd** [comment]
@jacogr any chance you could review this soon? As our crowdloan is live and the name, logo are not visible. Thank you.

**Comment by gonzamontiel** [comment]
> CI build is failing - https://github.com/polkadot-js/apps/runs/7128401099?check_suite_focus=true#step:3:738
> 
> Seems to be a file name mismatch.

should be fixed now

File: packages/apps-config/src/endpoints/productionRelayKusama.ts [hunk 1]
```diff
@@ -25,6 +25,14 @@ export const prodParasKusama: EndpointOption[] = [
       OnFinality: 'wss://altair.api.onfinality.io/public-ws'
     }
   },
+  {
+    info: 'amplitude',
+    homepage: 'https://pendulumchain.org/',
```
Comment (meinharrd) [inline packages/apps-config/src/endpoints/productionRelayKusama.ts:30]: @gonzamontiel Shouldn't this be 'https://pendulumchain.org/amplitude'?

Comment (gonzamontiel) [reply packages/apps-config/src/endpoints/productionRelayKusama.ts:30]:
```suggestion
    homepage: 'https://pendulumchain.org/amplitude',
```

Comment (gonzamontiel) [reply packages/apps-config/src/endpoints/productionRelayKusama.ts]: thanks, fixed : )

File: packages/apps-config/src/ui/logos/index.ts [hunk 2]
```diff
@@ -421,6 +423,7 @@ export const chainLogos = Object.entries({
 export const nodeLogos = Object.entries({
   'Acala Node': chainAcala,
   'Ajuna Node': nodeAjuna,
+  'Amplitude': chainAmplitude,
```
Comment (jacogr) [inline packages/apps-config/src/ui/logos/index.ts:426]:
```suggestion
  Amplitude: chainAmplitude,
```

This should make [CI happy](https://github.com/polkadot-js/apps/runs/7132831326?check_suite_focus=true#step:3:702). (Alternatively run `yarn lint` locally for auto-fixes)

**Review by jacogr — CHANGES_REQUESTED** [review]

**Review by jacogr — APPROVED** [review]

**Pull request merged**